test/db/cmd/cmd_af

```diff
@@ -15,6 +15,21 @@ EXPECT=<<EOF
 EOF
 RUN
 
+NAME=af display current offset function
+FILE=bins/elf/ls
+CMDS=<<EOF
+s main
+af
+afl.
+s. 1
```

Comment:
I think you should use sd and not s., Just to be sure and clearer

Reply:
Oh agree! did not noticed there was a comment on PR......
Thanks a lot for help!

Reply:
no problem ;)